lib/bus_kiosk_web/templates/kiosk/page.html.leex

```diff
@@ -30,12 +30,30 @@
     </div>
   <% end %>
 </div>
+<%= if @joined_stop_names do %>
+  <div phx-update="ignore">
+    <div class="add-stops" data-controller="add-stops"
+         data-add-stops-stop-ids="<%= @joined_stop_ids %>"
+         data-add-stops-stop-names="<%= @joined_stop_names %>"
+      >
+      <input data-target="add-stops.name" placeholder="<%= @joined_stop_names %>" />
```

Comment:
The placeholder text font makes it look like the input is already populated with data. Not a huge deal but the first few times I tried to highlight and edit the text that was there.